src/components/DatePicker.js

```diff
@@ -174,11 +174,17 @@ class DatePicker extends React.Component {
         style={[styles.component, styles.clearFix, this.props.style]}
         tabIndex='0'
       >
-        <div key='componentTop' style={[styles.componentTop, this.state.showCalendar && styles.componentTopOpen]}>
+        <div key='selectedDateWrapper' style={[
+          (this.props.selectedDateWrapperStyle || styles.selectedDateWrapper),
+          this.state.showCalendar && styles.selectedDateWrapperOpen
```

Comment:
Why don't we just add this into the array instead of using an "or" statement?
```
[
this.props.selectedDateWrapperStyle,
styles.selectedDateWrapper,
this.state.showCalendar && styles.selectedDateWrapperOpen
]
```